README.md

```diff
@@ -198,6 +225,24 @@ export const {StoreContext, useDispatch, useMappedState} = create<
 >();
 ```
 
+`create` takes an optional `options` object with the following options:
+- `defaultAreEqual` - the default implementation of `areEqual` to use in `useMappedState`, defaults to refence equality (`===`)
```

Comment:
`defaultAreEqual ` is a slightly weird prop name - perhaps `defaultEqualityChecker`?